src/cascadia/TerminalSettingsModel/init.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include <LibraryResources.h>
+
+// Note: Generate GUID using TlgGuid.exe tool
+TRACELOGGING_DEFINE_PROVIDER(
+    g_hTerminalAppProvider,
+    "Microsoft.Windows.Terminal.App",
+    // {24a1622f-7da7-5c77-3303-d850bd1ab2ed}

```

Comment:
you need to generate a new tracing guid and everything here.

Reply:
the guid is based on the name. you have to use tlgguid.exe to generate it. look it up on osgwiki.

Reply:
I meant to ask. I know we were discussing removing some of this from TSM. Should I just remove it all? Declare and hook up the new one? Or _porque no los dos_ it and introduce the new one but leave it unused?